packages/react/src/file-uploader/FileUploader.tsx

```diff
@@ -0,0 +1,31 @@
+import { FileUpload } from "@ark-ui/react";
+
+export const FileUploader = () => {
+  return (
+    <FileUpload.Root maxFiles={5}>
+      <FileUpload.Label>File Upload</FileUpload.Label>
+      <FileUpload.Dropzone>Drag your file(s) here</FileUpload.Dropzone>
+      <FileUpload.Trigger>Choose file(s)</FileUpload.Trigger>
+      <FileUpload.ItemGroup>
```

Comment:
let's skip this whole `ItemGroup` concept for now - we can add it in a later iteration